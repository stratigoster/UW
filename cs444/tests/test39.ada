PACKAGE foo IS
  procedure plus (a:integer;b:float);
  procedure plus (c:integer;d:float); -- this is alpha equivalent to the one above
END foo;
