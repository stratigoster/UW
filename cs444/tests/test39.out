***ERROR***:3:38:	Conflicting declaration `plus:function (integer,float) return void'

------
Compilation error
------
